File: csharp/core/Models/FileField.cs
using System.IO;

using Tea;

namespace AlibabaCloud.SDK.TeaFileform.Models
{
    public class FileField
    {
        [NameInMap("filename")]
        public string Filename { get; set; }

        [NameInMap("contentType")]
        public string ContentType { get; set; }

        [NameInMap("content")]
        public Stream Content { get; set; }
    }
}

File: php/src/FileField.php
<?php

namespace AlibabaCloud\Tea;

class FileField extends Model
{
    public $filename;
    public $contentType;
    public $content;

    public function __construct()
    {
        $this->_required = [
            "filename"    => true,
            "contentType" => true,
            "content"     => true
        ];
    }
}